bindata/etcd/restore-pod.yaml

```diff
@@ -24,6 +24,43 @@ spec:
         export ETCD_NAME=${NODE_NODE_ENVVAR_NAME_ETCD_NAME}
         export ETCD_INITIAL_CLUSTER="${ETCD_NAME}=https://${NODE_NODE_ENVVAR_NAME_ETCD_DNS_NAME}:2380"
         env | grep ETCD | grep -v NODE
+        export ETCD_NODE_PEER_URL=https://${NODE_NODE_ENVVAR_NAME_ETCD_DNS_NAME}:2380
+
+        # checking if data directory is empty, if not etcdctl restore will fail
+        if [ ! "$(ls -A /var/lib/etcd)" ]; then
```

Comment:
we only need to know that the dir exists right? restore will fail if /var/lib/etcd exists at all.

[ -d "/var/lib/etcd " ]

Reply:
confirmed, will change

```
[root@alpate-4jcgd-m-0 ~]# mkdir temp2
[root@alpate-4jcgd-m-0 ~]# ETCDCTL_API=3 ./assets/bin/etcdctl snapshot restore $BACKUP_FILE --name  "alpate-4jcgd-m-0.c.openshift-gce-devel.internal" --initial-cluster="alpate-4jcgd-m-0.c.openshift-gce-devel.internal=https://etcd-0.alpatel-alpatel-dr10-2020-02-24.gcp.devcluster.openshift.com:2380"              --initial-cluster-token openshift-etcd-0             --initial-advertise-peer-urls https://etcd-0.alpatel-alpatel-dr10-2020-02-24.gcp.devcluster.openshift.com:2380             --data-dir="./temp2"
2020-02-25 20:21:53.922765 I | pkg/netutil: resolving etcd-0.alpatel-alpatel-dr10-2020-02-24.gcp.devcluster.openshift.com:2380 to 10.0.0.3:2380
2020-02-25 20:21:53.923652 I | pkg/netutil: resolving etcd-0.alpatel-alpatel-dr10-2020-02-24.gcp.devcluster.openshift.com:2380 to 10.0.0.3:2380
Error: data-dir "./temp2" exists
[root@alpate-4jcgd-m-0 ~]#
```

Thanks, @hexfusion I was unaware of this feature.